lib/internal/js_stream_socket.js

```diff
@@ -157,6 +157,7 @@ class JSStreamSocket extends Socket {
     let pending = bufs.length;
 
     this.stream.cork();
+    // Prefer `var` over `let` for performance optemisation
```

Comment:
```suggestion
    // Prefer `var` over `let` for performance optimization
```

Reply:
AFAIK using let should not have any negative performance impact on any currently supported Node.js release line?

Reply:
@BridgeAR According to @mcollina [here](https://github.com/nodejs/node/pull/30380) it's still an issue on node v10.

Reply:
:+1: fixed, thanks